client/src/components/Map.js

```diff
@@ -1,14 +1,76 @@
-import { MapContainer, TileLayer} from "react-leaflet";
-
+import { MapContainer, TileLayer, Marker, Popup } from "react-leaflet";
+import { Link } from "react-router-dom";
+import ChangeView from "./ChangeView"
 
+//  need to add markers to map, change props.center
 function Map(props) {
 
   return (
     <MapContainer
       center={props.center}
-      zoom={13}
+      zoom={props.zoom}
       style={{ height: "600px", width: "600px" }}
     >
+      {/* ChangeView function allows map center to be dynamically changed */}
```

Comment:
This looks and works great. You could add comments to each marker but this is fine I know it's just repetitive